shared/src/components/exercise-preview/index.jsx

```diff
@@ -126,7 +126,10 @@ class ExercisePreview extends React.Component {
               actions={this.props.overlayActions}
               onClick={this.props.onOverlayClick} /> )}
           <div className="exercise-body">
-            <ExerciseBadges multiPart={info.isMultiPart} video={info.hasVideo} interactive={info.hasInteractive} writtenResponse={info.isWrittenResponse} />
+            {
+              this.props.questionType !== 'student-mpp' &&
+              <ExerciseBadges multiPart={info.isMultiPart} video={info.hasVideo} interactive={info.hasInteractive} writtenResponse={info.isWrittenResponse} />
+            }
             {!isEmpty(info.context) && !!this.props.isInteractive ? <ArbitraryHtmlAndMath className="context" block={true} html={info.context} /> : undefined}
```

Comment:
It's almost always better to have the component handle the logic instead of using inline logic via `{}`.  How about adding a `questionType` prop to the ExerciseBadges` which will return null if it's a `student-mpp`? https://github.com/openstax/tutor-js/blob/main/shared/src/components/exercise-badges/index.jsx#L117